p4-16/spec/P4-16-spec.mdk

```diff
@@ -3766,7 +3772,7 @@ cannot guess the user intent, so P4 requires the user to disambiguate.
 | `x << z`            | RHS of shift cannot be signed |  `x << (bit<8>)z ` |
 | `x < z`             | Different signs |  `X < (bit<8>)z ` |
 |                         |                           |  `(int<8>)x < z ` |
-| `1 << x`            | Width of `1` is unknown |  `32w1 << x ` |
+| `1 << x`            | RHS of int shift must be compile-time known |  `32w1 << x ` |
```

Comment:
Suggested fixes:
- Combine the two reasons, as an "either or"
- Write "None" for the alternative